Hacktoberfest/README.md

```diff
@@ -0,0 +1,19 @@
+# Hacktoberfest 2020
```

Comment:
https://github.com/kubernetes/community/tree/master/events 
I suggest putting the events in a dedicated dir.

Reply:
Makes sense. However, as we have ongoing activities such as High-Performance challenge program, this change may affect the accessibility of certain related files. How about we adjust globally when the activity is over @AndreMouche also suggested the same to me earlier.